ApollosProject/apollos-prototype
# Suppress Back Button Width

## DESCRIPTION

### What does this PR do, or why is it needed?

The back button on the SMS Verification screen filled the full width of the screen. This PR adds a view to suppress the width of the BackButton component.

### What design trade-offs/decisions were made?

### How do I test this PR?

1. On the login screen, enter your phone number
2. Horizontally aligned with the back button at the midpoint of the width of the screen, tap; nothing should happen

### What automated tests did you write?

## SCREENSHOTS

![Screen Recording 2019-12-02 at 10 21 AM](https://user-images.githubusercontent.com/29125070/69971180-80dd1a80-14ed-11ea-966f-1140c53b7f93.gif)

## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
